calendar/css/calendar.css

```diff
@@ -0,0 +1,66 @@
+
+.irnmn-calendar__weekday-header {
+    display: grid;
+    grid-template-columns: repeat(7, 1fr); /* 7 : days per row (one week) */
+    text-align: center;
+    padding: 0.5rem 0;
+    font-weight: bold;
+}
+
+.irnmn-calendar__weekday {
+    padding: 0.2rem;
+}
+
+/* Day Buttons Container Styling */
+.irnmn-calendar__days {
+    display: grid;
+    grid-template-columns: repeat(7, 1fr); 
+    gap: 0.2rem; /* Spacing between day buttons */
+    text-align: center;
+    margin-top: 0.5rem;
+}
+
+.irnmn-calendar__day-btn {
+    padding: 0.5rem;
+    background-color: #ffffff;
```

Comment:
@samuelsilvapt Should we make all the colours custom properties so they can be easily over written. I know it's not an issue if not using shadow DOM, but think it can be a nice approach to explore so we can see the limitations